### PR TITLE
Schema updates

### PR DESCRIPTION
**Pull request by rlskoeser**
Revise data package schemas to match schema revisions

Validation isn't passing on all files yet, but I think the field definitions are correct.

### CODE REVIEW

File: datapackage.json
Comment (kmcelwee) [inline datapackage.json:334]: While we're changing this, could we use consistent capitalization? (I think this one and the three above are the only ones that are different)

File: datapackage.json
Comment (kmcelwee) [inline datapackage.json:62]: Should we add 0-21 min-max constraint? Are there negative numbers?

File: datapackage.json
Comment (kmcelwee) [inline datapackage.json:102]: I think the pattern should be `"^CAe [1-9]\d{3}$"`The double backslash occurs in all the regex in this JSON. Perhaps because of how it was encoded?

**Review by kmcelwee — CHANGES_REQUESTED** [review]
I also looked more closely at the full documentation https://specs.frictionlessdata.io/table-schema/ and had some other recommendations that I now realize I should have addressed in the last PR:

* `"missingValues" = [""]` is the default that FrictionlessData uses, but you seem to use it in specific places. Is there a reason for that?

* We could use "type": "year" for fields like "Date Range Start" and "Date Range End" https://specs.frictionlessdata.io/table-schema/#year

And then if we ever return to this, it would be great to have each of these marked for all fields:
```
	...
	required: true/false
	unique: true/false
	description: "ported from google sheets"
	...
```

**Comment by rlskoeser** [comment]
@kmcelwee thanks for the helpful review and feedback! Responses:

- I generated these by inferring from the CSV files and then customizing, which I think includes the missing values declaration by default. It seems to me that it doesn't hurt to leave it in, and makes it more obvious if we ever want to customize and add other missing values (which we might). I checked and I think it's there for each sheet, but your comment made it sound like it was inconsistent — did you see something different?
- Thanks for suggesting `year`, will switch to that
- Defer to @elambrinaki on validation for earliest attestation 
- Agree that consistent capitalization would be good; I think it will be easier to make changes like that after this update is merged in, so someone can update the google sheet and then update the schema.
- backslashes in regexes have to be escaped or the json isn't valid, the double backslash has to be present for it to work (weird, I know)
- Agree it would be good to mark required and unique fields (not required and not unique are the defaults, so we only need to specify when they are). This probably wouldn't be very hard, and validating unique would be valuable. Should we make an issue for this and take a first stab at it if we have time?
- Your comment about the description field reminded me, there is optional high-level metadata we can include for the entire package; that seems like the right place to note that the CSVs are generated via sync from Google Sheets. Do you want to make a separate issue for that so we document it for future if we don't get to it immediately? Details: https://specs.frictionlessdata.io/data-package/#metadata

**Comment by rlskoeser** [comment]
@elambrinaki I apologize for requesting you to review this without telling you what I wanted from you! What I wanted was for you to review the data package definition I've revised that will be used to power the automatic validation so that you can check that I've added the fields and the constraints you wanted, but looking at it now I don't think this is a very useful way for you to review it. 

I think it will be easiest to get the validation passing after this pull request is merged in — probably a combination of changes in the spreadsheet and changes in the data package file. Kevin or I can show you how to view the details of what's failing validation, and it should be even easier to get to once we get Kevin's automated reporting merged in.

**Comment by kmcelwee** [comment]
> I checked and I think it's there for each sheet, but your comment made it sound like it was inconsistent — did you see something different?

No! I misread how it was nested. It's all good.

> Agree that consistent capitalization would be good; I think it will be easier to make changes like that after this update is merged in, so someone can update the google sheet and then update the schema.

- [x] Create issue

> Agree it would be good to mark required and unique fields (not required and not unique are the defaults, so we only need to specify when they are). This probably wouldn't be very hard, and validating unique would be valuable. Should we make an issue for this and take a first stab at it if we have time?

- [x] Create issue

> Your comment about the description field reminded me, there is optional high-level metadata we can include for the entire package; that seems like the right place to note that the CSVs are generated via sync from Google Sheets. Do you want to make a separate issue for that so we document it for future if we don't get to it immediately? Details: https://specs.frictionlessdata.io/data-package/#metadata

- [x] Create issue

**Review by kmcelwee — APPROVED** [review]

**Comment by elambrinaki** [comment]
@rlskoeser Got it, thank you! Please let me know when I can review the validation.

**Pull request merged**
